content/docs/1.6.4/deploy/important-notes/index.md

```diff
@@ -224,14 +224,14 @@ Longhorn v{{< current-version >}} does not allow the following:
 However, Longhorn still allows the listed actions when cloning or restoring volumes created with earlier Longhorn
 versions.
 
-### Backup Related Custom Resources Might Disappear
+### Backup Data On The Remote Backup Server Might Be Deleted
 
-Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+Longhorn will attempt to delete backup data on the remote backup server and clean up the backup-related custom resources (BackupVolume, BackupBackingImage, SystemBackup, and Backup) of the cluster in the following scenarios:
 
 - An empty response from the NFS server due to a brief server downtime.
-- A race condition between related Longhorn backup controllers.
+- There is a race condition that could potentially delete the remote backup volume and its corresponding backups when resetting the backup target within a short period.
 
-The backup information will be resynchronized during the next polling interval.  
```

Comment:
Need to briefly mention
- How we fix the issue and mention the ground truth is the backup data in the backup target
- After the fix, what users will see
  - backup, backup volume, ... will be deleted without deleting the backup data 
  - After the backup target is backup, the resources will be recreated

Reply:
Added. Thanks.

Reply:
Is this paragraph insufficient?

```txt
Starting with Longhorn v{{< current-version >}}:

- Longhorn does not delete the remote backup data during the synchronization.
- If the backup-related custom resources in the cluster are deleted unintentionally, they will be resynchronized from the remote backup server during the next polling interval.
```

Reply:
> Longhorn does not delete the remote backup data during the synchronization.

The statement is not clear, and users might misunderstand that Longhorn won't delete backup data anymore. `does not delete the remote backup data` happens when satisfying some conditions. As I mentioned in `How we fix the issue and mention the ground truth is the backup data in the backup target`, Need to clearly mention the conditions. 

> After the backup target is backup, the resources will be recreated

Then, it is the second part. `After the fix, what users will see....`

Reply:
> Is this paragraph insufficient?

No, need to update the paragraph which is confusing.

Reply:
Modified. Thanks